components/token.scss

```diff
@@ -0,0 +1,7 @@
+$fd-icons-path: "../scss/icons/";
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/components/token";
+@import "../scss/components/button";
```

Comment:
The token itself doesn't use any button styles so this file shouldn't be included.  That said, I see the "Clear All" button on the playground pages, but it shouldn't be there -- we should remove that.  The intent of the playground page is to display and style the component in question -- not display patterns or use cases within an application.